community/people/ew/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "person"
+title: "Eitan Weinstein"
+role: "Undergraduate Intern"
+image: "EitanHeadshot.jpg"
+email: "esweinstein2@wisc.edu"
+phone: "612-245-1556"
+city_state_zip: "Madison, WI 53706"
+services:
+  github: "eitan-weinstein"
+  facebook: "eitan.weinstein.98"
+  linkedin: "eitan-weinstein-792101224"
+ 
+ ---
```

Comment:
I also discovered an extra space at the beginning of this line that will be a problem! Subtle bug